zipline/finance/blotter/__init__.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .simulated_blotter import SimulatedBlotter
+from .blotter import Blotter
+from .core import (
+    load,
+    unregister,
+    register,
+    blotter_classes,
+)
+
+__all__ = ['SimulatedBlotter',
+           'Blotter',
+           'register',
+           'unregister',
+           'blotter_classes',
+           'load',
+           ]
```

Comment:
The trailing bracket here seems misaligned to me. I would find this easier to read as:
```python
__all__ = [
    'SimulatedBlotter',
    'Blotter',
    'register',
    'unregister',
    'blotter_classes',
    'load',
]
```